.travis.yml

```diff
@@ -27,6 +27,10 @@ matrix:
       python: 3.7
     - name: "3.8"
       python: 3.8
+    - name: "3.9"
+      python: 3.9-dev
```

Comment:
I'm more of a fan of nightly for failure allowed testing:
```yaml
    - python: nightly
  allow_failures:
    - python: nightly
```

Then we get signal on the net versions fixes sooner :)

Reply:
Is it so that `nightly` and `3.9-dev` are he same thing at the moment? And at some point in the future, `nightly` will be 3.10?

Sounds like a good idea to add it. 

I'd suggest also keeping `3.9-dev` as a gating check which can fail the whole build, ie. not in `allowed_failures`, to make sure new PRs pass on 3.9. (But only when the CPython bug is fixed and released.)